index.js

```
@@ -24,23 +24,49 @@
 
 var urlParse = require('url').parse;
 var clone = require('lodash.clonedeep');
+var async = require('async');
+var request = require('request');
+
+/*
+ * Definition of a 'convert()' facade function.
+ */
+var facade = function( ) {
+    
+  /*
+	 * check the type of the first argument if it's a string the return the
+	 * 'convertFromUrl' function - else return the default one
+	 */
+  if ( arguments.length === 0 )
+    throw "The convert() method expects at least 1 argument.";
```

Please throw an Error object.